### PR TITLE
(feat) - Update Pyth Oracle

### PR DESCRIPTION
**Pull request by aditya520**
**What changed? Why?**
Updated Pyth Oracle info.

**Notes to reviewers**

**How has it been tested?**

### CODE REVIEW

**Comment by cb-heimdall** [comment]
<h1>🟡 Heimdall Review Status</h1>
<table>
<tr>
<th>Requirement</th>
<th>Status</th>
<th>More Info</th>
</tr>
<tr>
<td>Reviews</td>
<td>
🟡
<code>0/2</code>
</td>
<td>
<details>
<summary>Denominator calculation</summary>
<table>
<tr>
<td>
<details>
<summary>Show calculation</summary>
<table>
<tr>
<td><code>1</code> if user is bot</td>
<td>0</td>
</tr>
<tr>
<td><code>1</code> if user is external</td>
<td>0</td>
</tr>
<tr>
<td>
From <code>.codeflow.yml</code>
</td>
<td>1</td>
</tr>
<tr>
<td>
Additional review requirements
<details>
<summary>Show calculation</summary>
<table>
<tr>
<td><strong>Max</strong></td>
<td><strong>0</strong></td>
</tr>
</table>
</details>
</td>
<td>0</td>
</tr>
<tr>
<td>
From <code>CODEOWNERS</code>
</td>
<td>0</td>
</tr>
<tr>
<td>Global minimum</td>
<td>0</td>
</tr>
<tr>
<td><strong>Max</strong></td>
<td>
<strong>
1
</strong>
</td>
</tr>
</table>
</details>
</td>
<td>1</td>
</tr>
<tr>
<td><code>1</code> if commit is unverified</td>
<td>1</td>
</tr>
<tr>
<td><strong>Sum</strong></td>
<td><strong>2</strong></td>
</tr>
</table>
</details>
</td>
</tr>
</table>

**Review by zencephalon — APPROVED** [review]

File: apps/base-docs/docs/tools/oracles.md
Comment (wbnns) [inline apps/base-docs/docs/tools/oracles.md:122]: s/the largest/a large/

File: apps/base-docs/docs/tools/oracles.md
Comment (wbnns) [inline apps/base-docs/docs/tools/oracles.md:127]: s/Most efficient/Efficient

File: apps/base-docs/docs/tools/oracles.md
Comment (wbnns) [inline apps/base-docs/docs/tools/oracles.md:132]: s/Base(Pyth Price Feeds)/Base (Pyth Price Feeds)/

File: apps/base-docs/docs/tools/oracles.md
Comment (wbnns) [inline apps/base-docs/docs/tools/oracles.md:138]: s/on the blockchain/onchain/

File: apps/base-docs/docs/tools/oracles.md
Comment (wbnns) [inline apps/base-docs/docs/tools/oracles.md:142]: s/Base(Pyth Entropy)/Base (Pyth Entropy)/

**Review by wbnns — CHANGES_REQUESTED** [review]
@aditya520 

Hi there! Thanks for the patience on this PR! Left a few comments / nits in-line, mainly related to a spacing typo and removal of superlatives.

File: apps/base-docs/docs/tools/oracles.md
Comment (wbnns) [inline apps/base-docs/docs/tools/oracles.md:122]: s/the largest/one of the largest/
s/blockchain/onchain/
s/on-chain/onchain/
s/the fastest on-chain/
s/one of the fastest onchain/

**Review by wbnns — CHANGES_REQUESTED** [review]
@aditya520 Thanks for the update! There are still a couple of superlatives and two remaining nits. Once fixed we can merge. 👍

File: apps/base-docs/docs/tools/oracles.md
Comment (wbnns) [reply apps/base-docs/docs/tools/oracles.md:122]: @aditya520 I still don't see the commit that made these changes, could you please update?

Comment (aditya520) [reply apps/base-docs/docs/tools/oracles.md]: Updated.

File: apps/base-docs/docs/tools/oracles.md
Comment (wbnns) [inline apps/base-docs/docs/tools/oracles.md:122]: s/is the one of/is one of/
/fastest onchain oracle/fastest onchain oracles/

**Review by wbnns — CHANGES_REQUESTED** [review]
@aditya520 

Thanks for the update. Looks like two typos need to be fixed and then I think we're good!

**Review by wbnns — APPROVED** [review]
ACK

**Comment by cb-heimdall** [comment]
Review Error for wbnns @ 2024-10-17 07:49:09 UTC
User cannot review their own commit

**Comment by brendan-defi** [comment]
@aditya520 this is failing our CI. Could you see if you can figure out what's up? Let me know if you're not able to determine / fix the issue.

Thanks!

**Comment by aditya520** [comment]
@brendan-defi Can you check again? I think it should work now.

File: apps/base-docs/docs/tools/oracles.md
Comment (brendan-defi) [inline apps/base-docs/docs/tools/oracles.md:140]: @aditya520 could you remove the duplicate parentheses around the link here? this will cause the link to break.

**Review by brendan-defi — APPROVED** [review]

**Comment by brendan-defi** [comment]
thanks @aditya520! approved and merging now

**Pull request merged**
